_config.yml

```diff
@@ -30,6 +30,9 @@ navigation:
   - text: Community
     url: /community/
     fa: fa fa-users fa-fw
+  - text: Meet our leaders
+    url: /meet-our-leaders/
+    fa: fa fa-user-tie fa-fw
```

Comment:
```suggestion
```

Reply:
completely removed from the _config.yml as it's used for navigation.

Reply:
Noted. I'm curious as to how we plan to navigate to the leadership page if it isn't included in the navigation links.